erdblick_app/app/feature.search.component.ts

```diff
@@ -32,25 +34,33 @@ import {KeyboardService} from "./keyboard.service";
                     <p-button (click)="stopSearch()" icon="pi pi-stop-circle" label="" [disabled]="!canPauseStopSearch"
                               pTooltip="Stop search" tooltipPosition="bottom"></p-button>
                 </div>
-                <div style="display: flex; flex-direction: row; justify-content: space-between; margin: 0.5em 0; font-size: 0.9em; align-items: center;">
+                <div style="display: flex; flex-direction: row; justify-content: space-between; font-size: 0.9em; align-items: center;">
                     <span>Elapsed time:</span><span>{{ searchService.timeElapsed }}</span>
                 </div>
-                <div style="display: flex; flex-direction: row; justify-content: space-between; margin: 0.5em 0; font-size: 0.9em; align-items: center;">
+                <div style="display: flex; flex-direction: row; justify-content: space-between; font-size: 0.9em; align-items: center;">
                     <span>Features:</span><span>{{ searchService.totalFeatureCount }}</span>
                 </div>
-                <div style="display: flex; flex-direction: row; justify-content: space-between; margin: 0.5em 0; font-size: 0.9em; align-items: center;">
+                <div style="display: flex; flex-direction: row; justify-content: space-between; font-size: 0.9em; align-items: center;">
                     <span>Matched:</span><span>{{ searchService.searchResults.length }}</span>
                 </div>
-                <div style="display: flex; flex-direction: row; justify-content: space-between; margin: 0.5em 0; font-size: 0.9em; align-items: center;">
+                <div style="display: flex; flex-direction: row; justify-content: space-between; font-size: 0.9em; align-items: center;">
                     <span>Highlight colour:</span>
                     <span><p-colorPicker [(ngModel)]="searchService.pointColor"
                                          (ngModelChange)="searchService.updatePointColor()" appendTo="body"/></span>
                 </div>
-                <p-accordion *ngIf="traceResults.length" class="trace-entries" [multiple]="true">
-                    <p-accordionTab [header]="trace.name" *ngFor="let trace of traceResults">
-                        <span>{{ trace.content }}</span>
-                    </p-accordionTab>
-                </p-accordion>
+
+                @for (message of diagnostics; track message; let first = $first) {
+                    <p-card class="diagnostics-message">
```

Comment:
1. Use one p-card with entries divided by separators.
2. Entries should be highlighted on hover.
3. Put the p-card up above everything if kept in this dialog panel (alternatively, put the entries into a [popover](https://primeng.org/popover) under the search bar and push it programmatically).